.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - run: npm ci
+      - run: npm run build
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: "3.5"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+      - name: Build Python dist
+        run: |
+          make clean
+          make build
+      - name: Upload Release Asset sdist
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ github.event.upload_url }}
+          asset_path: ./dist/imjoy-elfinder-${{ github.event.tag_name }}.tar.gz
```

Comment:
This assumes that the version and tag name are the same.

Reply:
Sounds reasonable.